README.md

```diff
@@ -1,39 +1,44 @@
 Using Python for Astronomical Data Analysis in the Era of JWST
 ================================================================
+
 The Space Telescope Science Institute and core developers from the Astropy community are sponsoring a workshop at the January 2017 meeting of the [American Astronomical Society](http://aas.org/) #AAS229 in Grapevine, Texas.
 
-Check out the [Schedule!](Schedule.md)
+Check out the [Schedule](Schedule.md)!
 
```

Comment:
why did you move the bang? I was using this example: https://help.github.com/articles/relative-links-in-readmes/